api-ref-burndown.py

```diff
@@ -111,14 +111,13 @@ def update_review_list(files, updated):
             what = "needs:%s" % update['tag']
             fdata[what] = update['number']
 
-
 for fname in sorted(glob.glob("%s/*.py" % TOP)):
     with open(fname) as f:
         fdata = {'filename': os.path.basename(fname)}
         content = f.readlines()
         done = True
         for key in PHASES:
-            if ".. %s\n" % key in content:
+            if "# %s\n" % key in content:
                 fdata[key] = "TODO"
                 done = False
                 counts[key].append(fname)
```

Comment:
Line 77 should probably be: `m = re.match('-# needs:(.*)', line)`

The api-ref source need ".." for comments, we need "#".